enterprise/migrations/0109_remove_use_enterprise_catalog_sample.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+
+
+from django.db import migrations
+
+from enterprise.constants import USE_ENTERPRISE_CATALOG
+
+
+def create_sample(apps, schema_editor):
+    """Create the `use_enterprise_catalog` sample if it does not already exist."""
+    Sample = apps.get_model('waffle', 'Sample')
+    Sample.objects.get_or_create(
+        name=USE_ENTERPRISE_CATALOG,
+        defaults={'percent': 0},
```

Comment:
We probably want the percentage at whatever it's at currently (100 I'm pretty sure) if the migration is reversed, right?